scifin/classifier/classifier.py

```diff
@@ -89,32 +91,42 @@ def dtw_distance(ts1, ts2, window=None):
     """
     
     # Checks
-    try:
-        assert(ts1.type=='TimeSeries' and ts2.type=='TimeSeries')
-    except TypeError:
-        raise TypeError("Series have to be of type TimeSeries.")
-        
+    if not isinstance(ts1, ts.TimeSeries) and not isinstance(ts2, ts.TimeSeries):
+        raise AssertionError("Series have to be of type TimeSeries.")
+    if not isinstance(mode, str) and not mode in ['abs', 'square']:
```

Comment:
for good coding practice, use `('abs', 'square')` instead so it is immutable (cannot be changed)

Reply:
Thanks for the tip.